Sources/SwiftKuery/Column.swift

```diff
@@ -32,6 +32,10 @@ In this example, a person `Table` class, containing three instances of the `Colu
  ```
  */
 public class Column: Field, IndexColumn {
+    // Property denoting whether default value is NULL
+    // If set to true a `nil` value for the `defaultValue` property will be interpreted as `NULL`
+    public let nullDefaultValue: Bool
```

Comment:
`nullDefaultValue` is public. If the comments above are doc comments, they'd need to start with `///`